UnityProject/Assets/Scripts/HealthV2/Living/PlayerHealthV2.cs

```diff
@@ -196,5 +199,166 @@ private void ApplyStartingAllergies()
 		}
 
 		#endregion
+
+		#region Electrocution
```

Comment:
I heavily dislike how hardcoded this is and much prefer the interface approach it had before when Nooney first PRed.